pennylane/transforms/qcut.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module provides the circuit cutting functionality that allows large
+circuits to be distributed across multiple devices
+"""
+
+from networkx import MultiDiGraph
+from pennylane.measure import MeasurementProcess
+from pennylane.operation import Operator, Tensor
+from pennylane.tape import QuantumTape
+
+
+def _add_operator_node(
+    graph: MultiDiGraph, op: Operator, order: int, wire_latest_node: dict
+) -> None:
+    """
+    Helper function to add operators as nodes during tape to graph conversion.
+    """
+    graph.add_node(op, order=order)
+    for wire in op.wires:
+        if wire_latest_node[wire] is not None:
+            parent_op = wire_latest_node[wire]
+            graph.add_edge(parent_op, op, wire=wire)
+        wire_latest_node[wire] = op
+
+
+def tape_to_graph(tape: QuantumTape) -> MultiDiGraph:
+    """
+    Converts a quantum tape to a directed multigraph.
+
+    Args:
+        tape (.QuantumTape): tape to be converted into a directed multigraph
```

Comment:
```suggestion
        tape (QuantumTape): tape to be converted into a directed multigraph
```
I think the dot prevents linking